client/menus/main.lua

```diff
@@ -138,7 +138,8 @@ menu.onClose = function()
 end
 
 lib.callback.register('customs:client:vehicleProps', function()
-    return QBCore.Functions.GetVehicleProperties(vehicle)
+    local vehicleProps = QBCore.Functions.GetVehicleProperties(vehicle)

```

Comment:
This doesn't need be changed, as the issue is solved with your changes in server.lua, this change is uneeded. I'd rather not have a variable that isn't used.